src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5PoNCreator.java

```diff
@@ -153,12 +182,21 @@ private static void writeTargetFactorNormalizeReadCountsAndTargetFactors(final F
                 .collect(Collectors.toList());
         try (final HDF5File file = new HDF5File(outFile, HDF5File.OpenMode.CREATE)) {
             final HDF5PoN pon = new HDF5PoN(file);
+            logger.info("Creating " + outFile.getAbsolutePath() + "...");
+            logger.info("Setting sample names ...");
             pon.setSampleNames(sampleNames);
+            logger.info("Setting target names ...");
             pon.setTargetNames(targetNames);
+            logger.info("Setting target factors (" + targetFactors.length + ") ...");
             pon.setTargetFactors(new Array2DRowRealMatrix(targetFactors));
+            logger.info("Setting normal counts (" + readCounts.counts().getRowDimension() +
```

Comment:
Be more explicit than `normal`.  Too much ambiguity when there are normalized counts of normals.